pkg/util/validation/exporter/ring.go

```diff
@@ -26,19 +27,27 @@ const (
 	// ringKey is the key under which we store the overrides-exporter's ring in the KVStore.
 	ringKey = "overrides-exporter"
 
-	// ringNumTokens is how many tokens each overrides-exporter should have in the ring.
-	// Overrides-exporters use a ring for tenant sharding and don't need perfect balancing.
-	ringNumTokens = 64
+	// ringNumTokens is how many tokens each overrides-exporter should have in the
+	// ring. Overrides-exporter uses timestamps to establish a ring leader, therefore
+	// only one token is needed.
+	ringNumTokens = 1
 
-	// ringAutoForgetUnhealthyPeriods is how many consecutive timeout periods an unhealthy instance
-	// in the ring will be automatically removed after.
-	ringAutoForgetUnhealthyPeriods = 4
+	// ringAutoForgetUnhealthyPeriods is how many consecutive timeout periods an
+	// unhealthy instance in the ring will be automatically removed after.
+	ringAutoForgetUnhealthyPeriods = 2
```

Comment:
I would still suggest 4 to let overrides-exporter or backend enough time to restart during a normal rollout.